definitions/git-repository/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: GIT
```

Comment:
Hi, we're pending to confirm if this is the domain we want to use with @jfjoly .

We'll likely prefer to use RES- (for resource, or entities that are not obviously linked to the ordinary ones that emit telemetry) or EXT- here (for the rest).

The rationale is that the domain field exists for legacy reasons so we'd prefer to avoid putting much semantics in the domain.  We should be able to clear this during today.

Reply:
Good to know! I will make this change!